about.html

```diff
@@ -132,5 +151,6 @@ <h4 class="title-events title-about">
         <p class="copy">© 2023. All Rights Reserved.</p>
       </section>
     </footer>
```

Comment:
- [ ] The `About` page footer should have a dark background footer, kindly make sure to change the background color of the footer on this page to a darker color so it looks identical to the provided design
- [ ] Also make sure to add some spacing between the logo and text so it looks cleaner

|Your website|Provided design|
|-|-|
|![Screenshot_20230322_090155](https://user-images.githubusercontent.com/25227022/226853102-fa00e990-ebc4-4e04-abec-98bcb9034b72.png)|![Screenshot_20230322_090400](https://user-images.githubusercontent.com/25227022/226853101-46004cd4-f2e8-46d0-8dc5-e8f304aa22b3.png)|